pip-requirements-doc

```diff
@@ -16,5 +16,3 @@ jplephem
 sphinx<1.7
 sphinx-astropy
 sphinx-gallery
```

Comment:
Are you sure that those are not needed for the doc build ? (added in #7443 by @Cadair )

Reply:
they are already present in the file above?

Reply:
They are duplicated within the *same file*. Is that necessary? I am not a `pip install` expert.

Reply:
oh, no they don't need to be duplicated. my question mark was for @saimn as I'm not he noticed that (but now reading back my comment, this intention doesn't come through)

Reply:
I didn't see that they were duplicated, sorry!